### PR TITLE
Algolia Search

### PR DESCRIPTION
**Pull request by philsturgeon**
Trying to figure out how to Algolia and the docs aren't quite complete here. Dunno if the functionality is done or not but I'm having a go!

I've done this so far, and dropped my algolia keys in, but nothing is happening. I've tried saving actions to trigger things. Do I need to configure indexes somehow, or will statamic try to create its own?

### CODE REVIEW

**Comment by philsturgeon** [comment]
Aha! Running `php please search:update` did it. Should I add that to the docs, to remind folks to do this if they want to see their stuff show up asap?

**Comment by jasonvarga** [comment]
Thanks Phil, good idea We have the "Updating Indexes" section, but it could be good to cross reference it from higher up the page.

File: content/collections/docs/search.md [hunk 1]
```diff
@@ -93,19 +93,25 @@ php please search:update name
...
 Algolia is a full-featured search and navigation cloud service. They offer search that is relevant and fast with results in under 100 ms (99% under 20 ms), that get prioritized and displayed using the customizable ranking formula.
 
+``` php
+'default' => [
+    'driver' => 'algolia',
+    'searchables' => 'all',
+],
+
+
 To set up the Algolia driver, create an account on [their site](https://www.algolia.com/), drop your API credentials into your `.env`, and install the composer dependency. Statamic will automatically create and sync your indexes.
```
Comment (philsturgeon) [inline content/collections/docs/search.md:115]:
```suggestion
To set up the Algolia driver, create an account on [their site](https://www.algolia.com/), drop your API credentials into your `.env`, and install the composer dependency. 

Statamic will automatically create and sync your indexes as you create a modify entries, but to get the initial index created on Algolia run `php please search:update`.
```

**Comment by philsturgeon** [comment]
There we go!

**Comment by jasonvarga** [comment]
Thanks!

**Pull request merged**